core/src/test/java/io/aiven/klaw/helpers/db/rdbms/TopicRequestsIntegrationTest.java

```diff
@@ -183,6 +238,262 @@ public void getTopicRequestsCountsForMyRequestsForTeam2() {
     assertThat(operationTypeCount.get(RequestOperationType.UPDATE.value)).isEqualTo(0L);
   }
 
+  @Test
+  @Order(4)
+  public void getAllRequestsFilteredByStatus() {
+
+    List<TopicRequest> james =
+        selectDataJdbc.selectTopicRequestsByStatus(
+            true, "James", "deleted", true, 101, null, null, null);
+    List<TopicRequest> james2 =
+        selectDataJdbc.selectTopicRequestsByStatus(
+            true, "James", "declined", true, 101, null, null, null);
+    List<TopicRequest> john =
+        selectDataJdbc.selectTopicRequestsByStatus(
+            true, "John", "created", true, 103, null, null, null);
+
```

Comment:
Can we add few topic requests based on OperationType (Claim/Create/Delete/Update) too